sys-libs/llvm-libunwind/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>zhanglei.april@gmail.com</email>
+		<name>Lei Zhang</name>
+	</maintainer>
+</pkgmetadata>
```

Comment:
Before I list a proxy-maint project here, do I need to ask for their permission?